Packs/ServiceDeskPlus_On_Premise/Integrations/ServiceDeskPlus_On_Premise/ServiceDeskPlus_On_Premise.yml

```diff
@@ -0,0 +1,801 @@
+      description: |-
+        Use this to get a list of tasks starting from this index.
+        e.g: 6
+      isArray: false
+      name: start_index
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Use this to mention the number of requests that needs to be returned.
+        e.g: 15. By default, will return only the first 10 requests.
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    - default: false
+      description: The column name and value to be searched for in the format of a JSON object. For example, {“subject”:“Change like this”,“priority.name”:“High”}.
+      isArray: false
+      name: search_fields
+      required: false
+      secret: false
+    - default: false
+      description: The name of the filter to use. For example, {“name”:“My_Open”}.
+      isArray: false
+      name: filter_by
+      required: false
+      secret: false
+    deprecated: false
+    description: View the details of requests. If no arguments are passed, details of all requests are returned.
+    execution: false
+    name: service-desk-plus-requests-list
+    outputs:
+    - contextPath: ServiceDeskPlus.Request.Requester
+      description: Who made the request.
+      type: Unknown
+    - contextPath: ServiceDeskPlus.Request.CreatedTime
+      description: The time the request was created.
+      type: Date
+    - contextPath: ServiceDeskPlus.Request.Template
+      description: The template that was used to create the request.
+      type: Unknown
+    - contextPath: ServiceDeskPlus.Request.DisplayId
+      description: The display ID of the request.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.Id
+      description: The unique ID of the request.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.Subject
+      description: The subject of the request.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.Technician
+      description: The technician that was assigned to the request.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.Status
+      description: The status of the request.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.DueByTime
+      description: The due date of the request.
+      type: Date
+    - contextPath: ServiceDeskPlus.Request.Group
+      description: The group to which the request belongs.
+      type: String
+    - contextPath: ServiceDeskPlus.Request.IsServiceRequest
+      description: Indicates whether the request is a service request.
+      type: Boolean
+    - contextPath: ServiceDeskPlus.Request.CancellationRequested
+      description: Indicates whether a cancellation was requested.
+      type: Boolean
+    - contextPath: ServiceDeskPlus.Request.HasNotes
+      description: Indicates whether the command has notes.
+      type: Boolean
+  - arguments:
+    - default: false
+      description: The ID of the request to delete.
+      isArray: false
```

Comment:
If it accepts a comma-separated list, this should be true

Reply:
Set isArray to true after verifying.